dbld/packages.manifest

```diff
@@ -18,6 +18,10 @@ unzip                           [centos, almalinux, fedora, debian, opensuse, rh
 # I dont know why this is needed.
 bzip2                           [centos, almalinux, fedora, debian, opensuse, rhel, rocky, ubuntu]
 
+# Compression test requirements (qpdf provides zlib-flate)
```

Comment:
as these are for tests in the dbld images, please move them to the end, like the other components required for the testing